Simple Inventory Management System/Inventory.cs

```diff
@@ -28,5 +28,40 @@ public void DisplayProducts()
             }
         }
 
+        public void EditProduct(string productName)
+        {
+            Product product = products.Find(p => p.Name == productName);
+            if (product != null)
```

Comment:
what should happen if the product is not found?  the method exits. You might want to inform the user that the product was not found.